.github/workflows/chromatic.yml

```diff
@@ -4,33 +4,42 @@ on:
   push:
     branches:
       - "master"
-  pull_request:
+  pull_request_target:
     types: [synchronize, labeled, unlabeled]
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref && github.ref || github.run_id }}
   cancel-in-progress: true
 
 jobs:
+  authorize:
+    environment:
+      ${{ github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository &&
+      'external' || 'internal' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - run: echo "true"
```

Comment:
What's the difference between just using `run: true` as in [the blog post](https://iterative.ai/blog/testing-external-contributions-using-github-actions-secrets)

Reply:
Linter was complaining that it doesn't comform to the GitHub schema.

Even the authors of that article use something similar in their own repo.